devel/parallel-hashmap/Portfile

```diff
@@ -16,11 +16,13 @@ maintainers             {@barracuda156 gmail.com:vital.had} openmaintainer
 description             A family of header-only, very fast and memory-friendly hashmap and btree containers
 long_description        {*}${description}
 homepage                https://greg7mdp.github.io/parallel-hashmap
-checksums               rmd160  2562cd557a276f73ac823465b4781d6bb2efc9ed \
-                        sha256  0cc203144321924cfbfcc401f42d8204c0dd24e2760c7a1c091baa16d9777c08 \
-                        size    2047800
+checksums               rmd160  a37046afe248c835966a3b66e5d242b5f3b8dd30 \
+                        sha256  4f462f51a3468166ea4cf87c80e001dc1999093264cf55cbda3492ca39a7730b \
+                        size    2059086 \
```

Comment:
Please remove that slash, otherwise we can miss it later. And an added extra line above `platfrom darwin` (no need for two empty lines).